gradle.properties

```diff
@@ -46,3 +46,4 @@ datadogApplicationId=do_not_change_here
 datadogClientToken=do_not_change_here
 datadogEnvironment=qa
 datadogServiceName=simple-android
+maestroTest=false
```

Comment:
Instead of maestroTest, I am thinking we can call it `isCI` or something like that so that it's much more clearer and flexible for us to use. WDYT?

Reply:
Suggestion: `signReleaseWithDebugKey` sounds like a good param here

Reply:
IDK, it's very specific IMO. Since it's a build param, I feel like it would be better to identify it as CI env or not make sense.

Reply:
Let's just go with `maestroTests` for now. We can change it in future if required.